dist/vl-grid.js

```diff
@@ -123,35 +123,43 @@ export class VlGrid extends NativeVlElement(HTMLDivElement) {
  */
 export class VlColumn extends NativeVlElement(HTMLDivElement) {
   static get _observedAttributes() {
-    return ['size', 'max-size', 'small-size', 'small-max-size', 'extra-small-size', 'extra-small-max-size', 'push'];
+    return ['size', 'max-size', 'medium-size', 'medium-max-size', 'small-size', 'small-max-size', 'extra-small-size', 'extra-small-max-size', 'push'];
   }
 
-  get _defaultMinSize() {
-    return 12;
+  connectedCallback() {
+    this._configureDefaults();
   }
 
   get _defaultMaxSize() {
     return 12;
   }
 
   get _sizeAttribute() {
-    return this.getAttribute('size') || this._defaultMinSize;
+    return this.getAttribute('size') || 8;
   }
```

Comment:
ik zou de methodes niet "...Attribute" noemen als je daarin ook een fallback steekt als dat attribute er niet is want dan is het niet dat attribute